client/components/PubReleaseDialog/PubReleaseDialog.tsx

```diff
@@ -91,20 +105,42 @@ const PubReleaseDialog = (props: Props) => {
 	};
 
 	const renderLatestReleaseInfo = (release) => {
-		const dateString = formatDate(release.createdAt, {
-			includeTime: true,
-			includePreposition: true,
-		});
 		const releaseUrl = pubUrl(communityData, pubData, { releaseNumber: release.branchKey + 1 });
+
 		return (
-			<p>
-				<a href={releaseUrl} rel="noopener noreferrer" target="_blank">
-					The most recent release
-				</a>{' '}
-				was made {dateString}. This will update the version of the Pub that readers see, but
-				they'll still be able to view previous releases by clicking the{' '}
-				<Icon icon="history" iconSize={12} /> History button on the Pub header.
-			</p>
+			<React.Fragment>
+				<p>
+					<Icon icon="history" iconSize={12} color="#888888" />{' '}
+					{' The previous Release ('}
```

Comment:
I think we should be able to omit the bracket/quotations from this line (and line 118).